apps/web/pages/auth/login.tsx

```diff
@@ -254,6 +258,24 @@ inferSSRProps<typeof getServerSideProps> & WithNonceProps<{}>) {
                     {t("signin_with_google")}
                   </Button>
                 )}
+                {isPasskeyLoginEnabled && (
+                  <Button
+                    color="secondary"
+                    className="w-full justify-center"
+                    disabled={formState.isSubmitting}
+                    StartIcon="key-round"
+                    onClick={async (e) => {
+                      e.preventDefault();
+                      await signInWithPasskey({
+                        tenantId: process.env.NEXT_PUBLIC_HANKO_PASSKEYS_TENANT_ID!,
```

Comment:
Forbidden non-null assertion needs an eslint comment to exclude or check to prevent

Reply:
I'll check for it explicitly